roles/common/tasks/apt.yml

```
@@ -0,0 +1,30 @@
+---
+
+- name: Update repositories
+  apt:
+    update_cache: yes
+
+- name: Install packages
+  apt:
+    state: present
```

Have the same result as in L3-5

```suggestion
    state: present
    update_cache: yes
```

---

done